Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Storage/Tabular/BookingCancellationReason.hs

```diff
@@ -40,6 +41,9 @@ mkPersist
       source Domain.CancellationSource
       reasonCode CancellationReasonTId Maybe
       additionalInfo Text Maybe
+      driverLat Double Maybe
```

Comment:
Here keys are find, only need to change in Domain as comment added above.

Reply:
pushed new changes.